demos/jans-tent/.deepsource.toml

```diff
@@ -0,0 +1,14 @@
+version = 1
```

Comment:
@moabu this is deep source, I would probably need to update `test_patterns` path if we may keep it.

Reply:
Yeah I don't think we can put this in . Instead we would use SonarCloud which jans is integrated with.